### DISCUSSION TITLE
I have a conflict. Do you have a idea?

### DISCUSSION THREAD

**Discussion by Nassir-Al** [neuefische/questions#278] [Web]
my PR:
https://github.com/Nassir-Al/capstone-project/pull/14

**Comment by ThomasF85** [comment]
Hi Nassir,

you have made a commit directly on the main branch while your feature branch had not been merged. This is causing this merge conflict now.
You have to fix the merge conflict before you can merge your feature branch into the main branch. Please do the following:
- Merge the main branch into the feature branch locally (`git merge main` when in the feature branch)
- Resolve the conflicts that are now appearing in the feature branch manually
- Make a new commit on the feature branch after the conflicts are resolved
- Push your commit to the remote
